app/validators/userValidator

```diff
@@ -0,0 +1,21 @@
+const passwordRegex = RegExp(/^(?=.*[a-z])(?=.*[A-Z])(?=.*[!@#$%^&*]).{9,}$/)
+
+module.exports.validateUser = function (user) {
+    if (user.email == null || user.email.trim().length === 0) {
+        return "E-Mail must be provided"
+    }
+
+    if (!passwordRegex.test(user.password)) {
+        return "Password must be more than 8 characters, contain 1 upper case letter, 1 lower case letter, and a special character"
```

Comment:
Just like on the API side - error should also inform about need of a number inside a password:
return "Password must be more than 8 characters, contain 1 upper case letter, 1 lower case letter, 1 number and a special character"